[prometheus] prometheus 2.4.3 upgrade, apply RFC3

predominant opened the pull request:
### Testing

```
hab studio enter
```

Run tests each in a new studio to be sure:

```
./prometheus/tests/test.sh
./prometheus2/tests/test.sh
```

Prometheus and Prometheus2 are at the same version.

thesentinels commented:
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

smacfarlane commented on prometheus2/tests/test.bats:
```diff
@@ -0,0 +1,25 @@
+}
+
+@test "Service is running" {
+  [ "$(hab svc status | grep "prometheus\.default" | awk '{print $4}' | grep up)" ]
```
This should be `prometheus2\.default 😄

predominant commented:
@smacfarlane `project.json` file is unknown to me, but included in release for prometheus1, as it was there before, I've removed it from `prometheus` and `prometheus2` thanks for the catch. Also fixed the test as requested.

predominant commented:
Squashed.

bdangit commented on prometheus/tests/test.bats.orig:
```diff
@@ -0,0 +1,32 @@
+  [ "$(command -v promtool)" ]
+}
+
+<<<<<<< HEAD
```
oh no.  looks like you need to resolve these conflicts

bdangit commented on prometheus/tests/test.sh.orig:
```diff
@@ -0,0 +1,37 @@
+hab pkg install core/busybox-static
+hab pkg binlink core/busybox-static ps
+hab pkg binlink core/busybox-static netstat
+<<<<<<< HEAD
```
more conflicts here but also I'm not sure if you intend to check in `test.sh.orig`

bdangit commented on prometheus2/tests/test.bats.orig:
```diff
@@ -0,0 +1,32 @@
+  [ "$(command -v promtool)" ]
+}
+
+<<<<<<< HEAD
```
more conflicts

bdangit commented on prometheus2/tests/test.sh.orig:
```diff
@@ -0,0 +1,37 @@
+hab pkg install core/busybox-static
+hab pkg binlink core/busybox-static ps
+hab pkg binlink core/busybox-static netstat
+<<<<<<< HEAD
```
more conflicts

bdangit commented on prometheus/hooks/run.orig:
```diff
@@ -0,0 +1,14 @@
+
+exec 2>&1
+
+<<<<<<< HEAD
```
conflicts but also `run.orig` is not a file you are intending to check in right?

predominant commented:
Re-squashed

bdangit commented:
LGTM
